src/components/TagList/TagList.spec.js

```diff
@@ -0,0 +1,37 @@
+import { mount } from '@vue/test-utils'
+import TagList from './TagList'
+import Tags from "@/components/Tags/Tags";
+
+describe('icon prop', () => {
+    test('passes an icon as a prop', () => {
+        const wrapper = mount(TagList, {
+            props: {
+                icon: 'pi-earth-line'
+            }
+        })
+
+        expect(wrapper.findAll('i').length).toEqual(1)
```

Comment:
I'm not sure this test is accurate. The `i` could be present without having the correct icon which I think is what this is testing?

Reply:
True! The same test is present in `Tag` component too, should I rewrite both?

Reply:
Probably a good idea!